docs/assets/img/docs/managing-pull-requests.drawio.svg

```diff
@@ -0,0 +1,498 @@
+                </foreignObject>
+                <text x="287" y="920" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                    Accept and wait...
+                </text>
+            </switch>
+        </g>
+        <path d="M 707 701 L 662.5 701 Q 657 701 657 695.5 L 657 672.5 Q 657 667 662.32 665.62 L 751.68 642.38 Q 757 641 757 646.5 L 757 695.5 Q 757 701 751.5 701 Z" fill="#ffffff" stroke="#000000" stroke-width="2" stroke-miterlimit="10" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 98px; height: 1px; padding-top: 677px; margin-left: 658px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 12px; font-family: system-ui-monospace,ui-monospace,Lucida Console,Menlo,monospace; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                <p>
+                                    brew
+                                    <br/>
+                                    pr-publish
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="707" y="681" fill="#000000" font-family="system-ui-monospace,ui-monospace,Lucida Console,Menlo,monospace" font-size="12px" text-anchor="middle">
+                    brew...
+                </text>
+            </switch>
+        </g>
+        <path d="M 527 531 L 550 471 L 604 471 L 627 531 Z" fill="#ffffff" stroke="#000000" stroke-width="2" stroke-miterlimit="10" transform="translate(0,501)scale(1,-1)translate(0,-501)" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe center; width: 98px; height: 1px; padding-top: 538px; margin-left: 528px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                Push the button
```

Comment:
Which button?

Reply:
```suggestion
                                Push the GitHub UI button
```

Reply:
@SMillerDev How about *Push the Merge button on GitHub?*

Reply:
Worth specifying to use `Rebase and merge` rather than just `Merge`?